src/Model/Join.php

```diff
@@ -182,6 +182,15 @@ static function (Model $entity) use ($name): self {
         );
     }
 
+    private function getModelTableString(Model $model): string
+    {
+        if (is_object($model->table)) {
```

Comment:
maybe better `$model->table instanceof Model`

Reply:
answered on Skype, `is_object` is fine now as it is the only possible object type code path, Expression(able) in table prop is not supported